libraries/AP_NavEKF3/AP_NavEKF3_feature.h

```diff
@@ -0,0 +1,13 @@
+/*
+  feature seclection for EKF3
+ */
+
+#pragma once
+
+#include <AP_Vehicle/AP_Vehicle_Type.h>
+
+// define for when to include all features
+#define EK3_FEATURE_ALL APM_BUILD_TYPE(APM_BUILD_AP_DAL_Standalone) || APM_BUILD_TYPE(APM_BUILD_Replay)
+
+#define EK3_FEATURE_WHEEL_ENCODER EK3_FEATURE_ALL || APM_BUILD_TYPE(APM_BUILD_Rover)
```

Comment:
Could I suggest some `#ifndef`s here?  Will be useful for people scrimping on space for their 1MB Rovers and whatnot.

Reply:
yep, will do